### ISSUE TITLE
Migrating required from JCenter

### ISSUE THREAD

**Issue by YiHaoHuang** [natario1/ZoomLayout#201]
### How do I?
JFrog Platform we will be sunsetting Bintray (including JCenter), GoCenter, and ChartCenter services on May 1st, 2021.
Any plan for distributing ZoomLayout via Maven Central?
https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

### Version used
latest version

**Comment by markusressel** [comment]
Yes #199 and yes #200 .

Please guys, keep calm. They will serve artifacts until the end of january 2022. There is *plenty* of time left to migrate.
I will keep this open so maybe only half as many people report the same thing...

**Comment by mman** [comment]
Safe for now except Android Studio is now emitting a warning and producing noise. #clean #build #please

```
Please remove usages of `jcenter()` Maven repository from your build scripts and migrate your build to other Maven repositories.
This repository is deprecated and it will be shut down in the future.
See http://developer.android.com/r/tools/jcenter-end-of-service for more information.
```

**Comment by markusressel** [comment]
I can't do much about this. @natario1 is the original project creator and package maintainer for this repo.
Maybe he can specify a rough timeframe when a migration will happen?

**Comment by Dark7SiD** [comment]
can u please move it

**Comment by mman** [comment]
Should stay open until addressed.

**Comment by markusressel** [comment]
Yes, this was unintented. Shouldn't happen again: #203

**Comment by markusressel** [comment]
@natario1 I guess labels with spaces dont work well with stalebot :cry:

**Comment by natario1** [comment]
Looks like long term is a flag (flag:long term) but we have is:long term in the config file! Anyway I marked it as enhancement which is already covered

**Comment by natario1** [comment]
Released 1.9.0 on MavenCentral.
